lib/src/tileset.dart

```diff
@@ -8,11 +9,17 @@ import 'themes/theme_layers.dart';
 class Tileset {
   final bool preprocessed;
   final Map<String, VectorTile> tiles;
+  late final ThemeLayerFeatureResolver resolver;
```

Comment:
is `late` needed?

Reply:
Yes, because we need to access `this` to initialize `resolver`. That means we cannot initialize before the constructor body.